run_tests.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+import unittest
+
+loader = unittest.TestLoader()
+tests = loader.discover(start_dir='.', pattern='*_test.py')
```

Comment:
I was thinking of this because it's easier to tweak in the future, but sure I can revert